.env

```diff
@@ -0,0 +1,3 @@
+MONGO_URL="mongodb://127.0.0.1:27017/wacurimvp"
+OPENTOK_KEY="46100042"
```

Comment:
Should we ignore this file?

Reply:
yep, but then wasn't sure how to share it.

Reply:
I'll grab it from here for now. Do you want to issue another PR so that this is omitted from the repo?

Also tested locally. Works great! 😸